python/lsst/daf/persistence/butlerLocation.py

```diff
@@ -211,15 +211,15 @@ def __repr__(self):
              self.additionalData, self.mapper, self.dataId)
 
     def __init__(self, pythonType, cppType, storageName, locationList, dataId, mapper, storage,
-                 usedDataId=None, datasetType=None):
+                 usedDataId=None, datasetType=None, additionalData=None):
```

Comment:
the init docstring (in the class docstring) needs to be updated with additionalData. (specifically I want to know, is what's passed in a dafBase.PropertySet?)